packages/Frontend/src/containers/AuthenticateContainer.jsx

```diff
@@ -23,47 +22,86 @@ class AuthenticateContainer extends Component {
     super(props);
     this.state = {
       email: '',
-      error: '',
+      error: false,
```

Comment:
Error handling was changed to match the way Google does it (which is much better I think), it'll either be false, or an object with a code and message.